proto/tendermint/farming/v1beta1/query.proto

```diff
@@ -17,33 +18,182 @@ service Query {
+                }
+            }
+        }
+        responses: {
+            key: "404"
+            value: {
+                description: "Bad Request"
+                examples: {
+                    key: "application/json"
+                    value: '{"code":5,"message":"rpc error: code = NotFound desc = plan plan_id not found","details":[]}'
+                }
+            }
+        }
+        responses: {
+            key: "500"
+            value: {
+                description: "Internal Server Error"
+                examples: {
+                    key: "application/json"
+                    value: '{"code":13,"message":"rpc error: code = Internal desc = error","details":[]}'
+                }
+            }
+        }
+    };
   }
 
   rpc Stakings(QueryStakingsRequest) returns (QueryStakingsResponse) {
     option (google.api.http).get = "/cosmos/farming/v1beta1/stakings/{farmer}";
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+            description: "Returns a list of all stakings that corresponds to the farmer";
+            external_docs: {
+                url: "https://github.com/tendermint/farming/tree/master/docs/How-To/cli#stakings";
+                description: "Find out more about the query and error codes";
+            }
+        responses: {
+            key: "400"
+            value: {
+                description: "Bad Request"
+                examples: {
+                    key: "application/json"
+                    value: '{"code":3,"message":"rpc error: code = InvalidArgument desc = empty request","details":[]}'
+                }
+            }
+        }
+      };
   }
 
   rpc TotalStakings(QueryTotalStakingsRequest) returns (QueryTotalStakingsResponse) {
     option (google.api.http).get = "/cosmos/farming/v1beta1/total_stakings/{staking_coin_denom}";
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      description: "Returns total stakings that corresponds to the staking_coin_denom";
+          external_docs: {
+              url: "https://github.com/tendermint/farming/tree/master/docs/How-To/cli#totalstakings";
+              description: "Find out more about the query and error codes";
+          }
+      responses: {
+          key: "400"
+          value: {
+              description: "Bad Request"
+              examples: {
+                  key: "application/json"
+                  value: '{"code":3,"message":"rpc error: code = InvalidArgument desc = empty request","details":[]}'
+              }
+          }
+      }
+    };
   }
 
   rpc Rewards(QueryRewardsRequest) returns (QueryRewardsResponse) {
     option (google.api.http).get = "/cosmos/farming/v1beta1/rewards/{farmer}";
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      description: "Returns a list of rewards that corresponds to the farmer";
```

Comment:
Also, we are returning `sdk.Coins` for rewards: https://github.com/tendermint/farming/blob/1b9d82a7ab70ffd1223acc0abe5b2dc9d20843a1/x/farming/keeper/grpc_query.go#L233-L235

Reply:
If the above proposal is okay to go, then the description for this will be:
- Returns a list of reward coins that corresponds to the farmer